README.md

```diff
@@ -1,75 +1,49 @@
 
+## Project web demo link. 
 
-## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+Built With Html. Css. JS.
 
-## Live Demo (if available)
+Website link: 
 
-[Live Demo Link](https://livedemo.com)
+# Getting starting 
+To get a local copy up and running follow these simple example steps:
 
+## Prerequisites
+ Vscode. 
+ must have git installed.
 
-## Getting Started
+## Setup 
+Clone this repo with:
+https://github.com/eerapheal/toDolist
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+cd awesomebook-ES6
 
+## Install 
+npm install
 
-To get a local copy up and running follow these simple example steps.
+Author1 Ekpenisi Erue Raphael.
 
-### Prerequisites
+GitHub: (https://github.com/eerapheal); 
+Twitter: (https://twitter.com/ekpenisiraphael); 
+LinkedIn: (https://www.linkedin.com/in/ekpenisi-e-raphael-9678a221a/)
 
-### Setup
+# 🤝 Contributing
+ Contributions, issues, and feature requests are welcome! Feel free to check the [issues page]Fork the Project Create your Feature Branch (git checkout -b 'branchname') Commit your Changes (git commit -m 'Add some branchname') Push to the Branch (git push origin branchname) Open a Pull Request Feel free to check the
 
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
+ ## Deployment
+  Push changes to a branch. Wait for the build to pass on our CI server. Tell Hubot to deploy it. Verify that the changes work and fix any problems that come up. Merge the branch into master.
 
+## Show your support 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
+## Acknowledgments 
+MICROVERSE and My Team.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+License This project is MIT licensed.
```

Comment:
- [ ] OPTIONAL Please, consider giving here the link to your MIT.md file, to make your README more organized and professional.